aiida_quantumespresso/cli/calculations/pw2wannier90.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+Command line scripts to launch a `Pw2WannierCalculation` for testing and demonstration purposes.
+
+This launcher assumes that the SEED and the PREFIX used in the previous PW calculation (parent_folder)
+are the same as those hardcoded in the Pw2WannierCalculation class.
+We also hardcode some parameters and options.
+"""
+from __future__ import absolute_import
+import click
+
+from aiida.cmdline.params import options, types
+from aiida.cmdline.params.options import OverridableOption
+from aiida.cmdline.utils import decorators
+
+from aiida_quantumespresso.cli.utils import options as options_qe
+
+PARENT_FOLDER = OverridableOption(
```

Comment:
These will most likely be reusable so maybe we can put them in `aiida_quantumespresso.cli.utils.options`?

Reply:
I understand that the short option names should be unique between all options defined in `cli.utils.options`, so I didn't want to add any more. For instance, `-S` for single_file would clash with `-S` for smearing (or `-s` for structure). If this is not a problem I can move them there, sure.

Reply:
Fair point, but I would say that the `smearing` is way more specific and so the single file data might take precedence. In any case, for now it is not that important. As soon as we have another launcher that requires the same option, we can centralize it. Also, although sometimes useful, a short option is not necessary. For less common options like `smearing` we could just decide to only provide the long variant so there won't be a clash

Reply:
...or just accept that they won't be able to be used together?

Reply:
On second thought, no, that's not a great idea. Ok thanks.